atomics/T1059.004/T1059.004.yaml

```diff
@@ -243,3 +243,13 @@ atomic_tests:
     elevation_required: false
     command: |
       uname -srm
+
+- name: emacs spawning an interactive system shell
+  description: |
+    emacs can be used to break out from restricted environments by spawning an interactive system shell. Ref: https://gtfobins.github.io/gtfobins/emacs/  
+  supported_platforms:
+  - linux
+  executor:
```

Comment:
**todo:** Can you add prereqs for installing emacs ?

Reply:
yep, sure

Reply:
Added prereqs